NoteworthyUserManual.md

```diff
@@ -0,0 +1,46 @@
+# Description 
+Welcome to Noteworthy! We are the #1 Notetaking Application For Markdown files. Some of our current available functions >>> include writing, viewing, searching, listing, deleting, clearing, and converting to PDF. If you have any questions relating to Markdown the link below provides useful information
+[Markdown Guide](https://www.markdownguide.org/)
+
+# Set Up/Installation 
+You can access the required code for the Noteworthy application using the link at the bottom of this section. Furthermore, it is worth noting that this is a command line application so no external downloads are required. Finally, you need to ensure you have a "Notes" subfolder inside of your Noteworthy directory.
+[Noteworthy Code](https://github.com/augustus-thomas/noteworthy)
+
+# Current Functions
+### Home Page 
+The purpose of the homepage is to allow you to access all the available functions for the application. For example, if you want to add notes to a file then you would type "add" while on the homepage. Furthermore, if you wish to exist the app this can also be done through the home page. 
```

Comment:
"exit" not "exist"

Reply:
here you don't use the "when prompted to specify an action you wish to perform" clause and it still reads extremely clearly. imo we dont need that clause in all the other sections